Heejung/Lv2/프린터.js

```diff
@@ -0,0 +1,26 @@
+function solution(priorities, location) {
+  let answer = 0;
+  let myDoc = location; // 내가 요청한 문서 위치
+
+  // 다 출력되면 반복문 탈출
+  while (priorities.length > 0) {
+    let shiftDoc = priorities.shift(); // 맨 앞 문서
+
+    // 맨 앞 문서보다 우선순위 높은 문서 있으면 자리 변경
+    if (priorities.filter((priority) => priority > shiftDoc).length) {

```

Comment:
```suggestion
    if (priorities.some((priority) => priority > shiftDoc)) {
```
여기서는 filter로 새로운 배열을 생성해서 length를 판단하는 용도로 짜는것보다는 some이라는 메서드를 쓰는게 성능면에서 더 좋을거같아요.